qac_compile_commands.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+# This script turns the -isystem flag into the -I flag.
+# This is needed for Helix QAC to work properly.
+# Usage: qac_compile_commands <REFRESH_COMPILE_COMMANDS>
+
+REFRESH_COMPILE_COMMANDS=$(realpath $1)
```

Comment:
I don't think using realpath is necessary here.

Since we make the compile_commands target a data dependency, the python script should be copied into the runtime environment.

(I tested this locally and it seemed to work).

Reply:
I needed it in the previous version of this script but, now it is not necessary.